src/map/pc.c

```diff
@@ -4392,10 +4392,11 @@ int pc_payzeny(struct map_session_data *sd,int zeny, enum e_log_pick_type type,
 /*==========================================
  * Cash Shop
  *------------------------------------------*/
-
+//Changed Kafrapoints calculation. [Normynator]
 int pc_paycash(struct map_session_data *sd, int price, int points)
 {
 	int cash;
+	int smp;
```

Comment:
smp means store-minus-points, just a variable to decrease kafrapoints later at line 4228